### DISCUSSION TITLE
Entry control in .NET MAUI Preview 8 appends text at beginning

### DISCUSSION THREAD

**Discussion by mahebist** [dotnet/maui#2628] [Q&A]
I am using Entry control in .NET MAUI Preview 8 . When I enter any character it appends text at beginning.

If I type Mango then it type ognaM. What may the reason for this behaviour.

<Entry Text="{Binding LoginModel.UserName}" Placeholder="Enter User Name" WidthRequest="400" HorizontalOptions="FillAndExpand"></Entry>

If I use Editor then typing is fine.
<Editor Text="{Binding LoginModel.UserName}" Placeholder="Enter User Name" WidthRequest="400" HorizontalOptions="FillAndExpand"></Editor>

**Comment by hartez** [comment]
Fixed by PR #2638

**Reply by mahebisht** [reply]
How should I fetch this update.
